build.gradle

```diff
@@ -40,7 +42,7 @@ allprojects {
 
         dependencies {
             dependency 'org.projectlombok:lombok:1.16.8'
-            dependency 'com.mesosphere:marathon-client:0.3.0'
+            dependency 'com.mesosphere:marathon-client:0.3.1-SNAPSHOT'
```

Comment:
Using com.mesosphere:marathon-client:0.3.1-SNAPSHOT (built locally) to support DC/OS secrets in App.env